repose-aggregator/components/filters/valkyrie-authorization-filter/build.gradle

```diff
@@ -7,13 +7,15 @@ dependencies {
     compile "com.typesafe.scala-logging:scala-logging-slf4j_$scalaMajDotMin"
     compile "org.openrepose:http-delegation"
     compile "com.typesafe.play:play-json_$scalaMajDotMin"
-    compile "com.josephpconley:play-jsonpath_$scalaMajDotMin"
+    compile("com.josephpconley:play-jsonpath_$scalaMajDotMin") {
+        transitive = false
```

Comment:
what came in to force this?

Reply:
The way they declared one of their dependencies was causing Gradle to choke and not be able to find it.

Reply:
The one that was causing the issue was:

- `com.typesafe.play:play-json_2.11:[2.5.0,2.5.+)`